.github/release-drafter.yml

```diff
@@ -27,9 +27,15 @@ autolabeler:
   - label: 'java-sdk'
     title:
       - '/.*\(.*java.*\):/i'
+    files:
+      - '/sdk/java-sdk/**'
   - label: 'scala-sdk'
     title:
       - '/.*\(.*scala.*\):/i'
```

Comment:
can we remove the title sections now?

Reply:
Let's see if it kicks in with the file globs. Something else might be broken.